Projects/GLU/GLU07/GLU07-AWS100.md

```diff
@@ -0,0 +1,54 @@
+
+# GLU07-AWS100 - Setup a simple state machine with at least 2 steps
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
+## Estimated Time
+- 60 minutes 
+
+## Estimated Cost
+- The Step Functions free tier includes 4,000 free state transitions per month. After the free tier exhausts you pay $0.025 per 1,000 state transitions. See Details [here](https://aws.amazon.com/step-functions/pricing/)
+- The AWS Lambda free usage tier includes **1M free requests per month and 400,000 GB-seconds of compute time per month**
+
+## Project's Author(s)
+
+- [Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+- Create an AWS Lambda to add 2 numbers supplied as input -[ sample code for step 1](./GLU07-AWS100-SF-step1.py?raw=true)  
+- Create an AWS Lambda to return the square of a number-  [ sample code for step 2](./GLU07-AWS100-SF-step2.py?raw=true)  
+- Create a state machine to connect the above two Lambdas
+	-go to the step  functions console click on create state machine , do not chnage the default settings.
+	-copy [ this ](./GLU07-AWS100-SF-step2.py?raw=true), template to paste into Definition but carefully replace Lambda ARNs with the Lambdas created in above steps where necessary in the template. 
```

Comment:
- Inconsistencies in the bullet points and dashes (` - `), as well as capitalization
- Typo: `change`
- `- copy this template` does not need a comma before the template

Reply:
Fixed